x-pack/plugins/global_search_bar/kibana.jsonc

```diff
@@ -11,7 +11,8 @@
       "global_search_bar"
     ],
     "requiredPlugins": [
-      "globalSearch"
+      "globalSearch", 
+      "fileUpload",
```

Comment:
I'm a bit reluctant to introduce this plugin dependency. We are using it to call the API `fileUpload.getMaxBytes()` which returns the Max **File** size allowed.

AFAIK, the limit that we want to impose here is different. It's about the number of characters to search for in the Global Search Bar.

I guess my question is: what's the purpose of limiting the number of characters? Is it driven by a browser/HTTP limitation? Do we know that hard limit? Should we apply that as the limit?

Apologies if these questions are resolved elsewhere 😇

Reply:
This PR might not be an issue based on a slack conversation with @yuliacech where the issue with global search was not reproducible in her testing

Reply:
>  We are using it to call the API fileUpload.getMaxBytes() which returns the Max File size allowed. AFAIK, the limit that we want to impose here is different. It's about the number of characters to search for in the Global Search Bar

Came here to ask that exact same question. Adding an hardcoded arbitrary limit would be totally fine to me if we need so, but I really don't get why we're using that dep toward the fileUpload plugin tbh.

Reply:
Using char limit in https://github.com/elastic/kibana/pull/186312 thanks!